articles/azure/azs-how-create-vm-image-portal.md

```diff
@@ -0,0 +1,127 @@
+---
+title: How to create a custom image from a VM on Azure Stack Hub using the UKCloud Azure Stack Hub portal
+description: Provides details on how to create a custom image from a VM using the portal
+services: azure-stack
+author: Daniel Brennand
+reviewer:
+lastreviewed: 03/04/2020
+
+toc_rootlink: Users
+toc_sub1: How To
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Create a custom image from a VM - Portal
+toc_fullpath: Users/How To/azs-how-create-vm-image-portal.md
+toc_mdlink: azs-how-create-vm-image-portal.md
+---
+
+# How to create a custom image from a VM on Azure Stack Hub using the UKCloud Azure Stack Hub portal
+
+## Overview
+
+You can create an image resource from a generalised virtual machine (VM) that is stored as either a managed disk or an unmanaged disk in a storage account. You can then use the image to create multiple VMs.
+
+This article explains how to create a custom image from a VM on Azure Stack Hub, which you can then use to deploy other VMs.
+
+## Prerequisites
+
+To complete the steps in this article, you must have appropriate access to a subscription in the Azure Stack Hub portal.
+
+## [1. Generalise your VM](#tab/tabid-1)
+
+> [!WARNING]
+> Once you've generalised a VM, you **cannot** log back into it.
+
+## [2. Creating the image](#tab/tabid-2)
+
+> [!WARNING]
+> Capturing a VM image will make it unusable and **cannot** be undone.
+
+1. Log in to the [Azure Stack Hub portal](https://portal.frn00006.azure.ukcloud.com/).
+
+    For more detailed instructions, see [Getting Started Guide for UKCloud for Microsoft Azure](azs-gs.md).
+
+2. In the *favorites* panel, select **Virtual machines**.
+
+3. In the *Virtual machines* blade, select the VM that you want to capture.
+
+4. In the top menu, click **Capture**.
+
+    ![Capture VM image](images/azs-browser-capture-vm.png)
+
+5. In the *Create image* blade, provide a **Name** for the new image.
+
+6. From the **Resource group** list, select an existing resource group, or create a new one by clicking the **Create new** link and then typing a name for your new resource group in the pop out window.
```

Comment:
I changed this to match how it's written in other articles. Might want to quickly check to confirm it's actually how it works

Reply:
Hi @suehighmoor. This is correct 😄 👍 thanks!